Modify the motd command to only send to unit users (#1552)
ScilifelabDataCentre/dds_web: SPRINTLOG.md

- Flask command to update unit quotas ([#1551](https://github.com/ScilifelabDataCentre/dds_web/pull/1551))
- Bump python base image to 3.12 and related libraries in both web and client([#1548](https://github.com/ScilifelabDataCentre/dds_web/pull/1548))
- Warning_level option defaults to 0.8([#1557](https://github.com/ScilifelabDataCentre/dds_web/pull/1557))

# 2024-09-24 - 2024-10-04

> - Modify the motd command to only send to unit users([#1552](https://github.com/ScilifelabDataCentre/dds_web/pull/1552))

Reviewer: ```suggestion
- Add option to motd command for sending to unit users only([#1552](https://github.com/ScilifelabDataCentre/dds_web/pull/1552))
```
